controllers/operands/kubevirt_test.go

```diff
@@ -2655,6 +2655,37 @@ Version: 1.2.3`)
 			})
 		})
 
+		Context("VM state storage class", func() {
+			It("should modify storage class according to HCO CR", func() {
+				existingResource, err := NewKubeVirt(hco)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Modify HCO's eviction strategy configuration")
```

Comment:
Please fix to match the actual modification

Reply:
:facepalm: nice catch